From 0fa9753b6df677e02ade15fe06226adf0988aef5 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Sun, 2 Aug 2020 20:10:41 +0200
Subject: [PATCH] fix build with (system-)cppunit 1.14.0

by #include <unotest/bootstrapfixturebase.hxx> so that CPPUNIT_TEST_FIXTURE
is defined

Change-Id: Iacbfb2f513df6e7eb7ca04f7900cd4e6b3fd69c7
---
 svx/qa/unit/XTableImportExportTest.cxx | 1 +
 svx/qa/unit/xoutdev.cxx                | 1 +
 2 files changed, 2 insertions(+)

diff --git a/svx/qa/unit/XTableImportExportTest.cxx b/svx/qa/unit/XTableImportExportTest.cxx
index 9356f2691323..31962a52be52 100644
--- a/svx/qa/unit/XTableImportExportTest.cxx
+++ b/svx/qa/unit/XTableImportExportTest.cxx
@@ -10,6 +10,7 @@
 #include <cppunit/TestAssert.h>
 #include <cppunit/TestFixture.h>
 #include <cppunit/extensions/HelperMacros.h>
+#include <unotest/bootstrapfixturebase.hxx>
 
 #include <sal/types.h>
 #include <sfx2/app.hxx>
diff --git a/svx/qa/unit/xoutdev.cxx b/svx/qa/unit/xoutdev.cxx
index 4b767ca1481e..0b5148b393d5 100644
--- a/svx/qa/unit/xoutdev.cxx
+++ b/svx/qa/unit/xoutdev.cxx
@@ -12,6 +12,7 @@
 #include <cppunit/TestAssert.h>
 #include <cppunit/TestFixture.h>
 #include <cppunit/extensions/HelperMacros.h>
+#include <unotest/bootstrapfixturebase.hxx>
 
 #include <sal/types.h>
 #include <sfx2/app.hxx>
-- 
2.20.1

